gcp_variant_transforms/libs/bigquery_row_generator_test.py

```diff
@@ -154,14 +155,35 @@ def _get_table_schema(self):
     schema.fields.append(call_record)
     return schema
 
+  def _make_header(self, header_num_dict):
+    # type: (Dict[str, str]) -> VcfHeader
+    """Builds a VcfHeader based on the header_num_dict.
+
+    All fields of parser._Info are set to their default values except for the
+    'num' which is set using the values of the header_num_dict based on the
+    mapping defined in parser.field_counts.
+
+    Args:
+      header_num_dict: a dictionary mapping info keys to string num values.
+    """
+    infos = {}
+    for k, v in header_num_dict.iteritems():
+      if v in parser.field_counts:
+        pyvcf_num_field_value = parser.field_counts[v]
+      else:
+        pyvcf_num_field_value = int(v)
+      infos[k] = parser._Info(id=None, num=pyvcf_num_field_value,
```

Comment:
nit: I realize that `id` is not used here, but it's probably more intuitive to set it to `id=k`.